packages/client/components/ReflectionCard/EmojiUsersReaction.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import {EmojiUsersReaction_reactji$key} from '~/__generated__/EmojiUsersReaction_reactji.graphql'
+import {PALETTE} from '../../styles/paletteV3'
+
+const EmojiUsersReactionRoot = styled('div')({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  maxHeight: 300,
+  maxWidth: 280,
+  overflow: 'auto',
+  wordWrap: 'break-word',
+  overflowWrap: 'break-word',
+  whiteSpace: 'normal'
+})
+
+const DarkerGrayPart = styled('span')({
+  color: PALETTE.SLATE_400
+})
+
+const LIST_FORMATTER =
+  Intl.ListFormat !== undefined
+    ? new Intl.ListFormat('en', {style: 'long', type: 'conjunction'})
+    : {format: (arr: string[]) => arr.join(', ')} // fallback for safari pre Big Sur
+
+interface Props {
+  reactjiRef: EmojiUsersReaction_reactji$key
+  reactjiShortName?: string
+}
+
+const EmojiUsersReaction = ({reactjiRef, reactjiShortName}: Props) => {
+  const reactji = useFragment(
+    graphql`
+      fragment EmojiUsersReaction_reactji on Reactji {
+        id
+        users {
+          id
+          preferredName
+        }
+      }
+    `,
+    reactjiRef
+  )
+
+  return (
+    <EmojiUsersReactionRoot>
+      {LIST_FORMATTER.format(reactji.users.map(({preferredName}) => preferredName))}
```

Comment:
+1 Possible improvement: I like how Slack etc. replace the viewers name with "You".
![Screenshot from 2022-10-04 17-16-12](https://user-images.githubusercontent.com/7331043/193858720-15116fe5-1edb-4a43-896d-c3267ebf0724.png)

Reply:
good suggestion! for now, I'd want to avoid introducing additional complexity and land it as is. I'll create a task for that!